### PR TITLE
Create a pull request template

### PR DESCRIPTION
**Pull request by Thlumyn**
Adds a template for pull requests

This means that when people create a pull request, the description section comes pre-filled with some headers and comments.

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+## Why This Is Good For ClanGen
+
+<!-- If this is a bugfix, you can remove this section. -->
+<!-- Please add a short description of why you think these changes would benefit the game. -->
+<!-- If this PR was the result of discussion/testing on the discord, please add a link to the discord conversation here. -->
+<!-- If you have multiple features that can stand on their own, or unrelated bugfixes, please create separate PRs for them. -->
+
+## Linked Issues
+
+<!-- If this is not a bugfix, you can remove this section. -->
```
Comment (scribblecrumb) [inline .github/PULL_REQUEST_TEMPLATE.md]: We do have some issues related to enhancements and not bugfixes, it would be good to adjust the wording here to account for it.  perhaps to: "If this is not related to any issues, you can remove this section.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+<!-- If this was in response to a github issue, please write it here with the format Fixes: #1234 so that github knows to link the issues. -->
+<!-- If this PR was the result of discussion/testing on the discord, please add a link to the discord conversation here. -->
+
+## Proof of Testing
+
+<!-- Include any screenshots or debugging steps here. At least one form of proof of testing is REQUIRED for all new content. You must be able to run the code locally before you PR it here. -->
```
Comment (scribblecrumb) [inline .github/PULL_REQUEST_TEMPLATE.md]: I would add something about linking testing threads here as well, if one exists.  Perhaps change the first sentence to "Include any screenshots, debugging steps, or links to beta testing threads here."

**Review by scribblecrumb — CHANGES_REQUESTED** [review]

**Comment by Thlumyn** [comment]
@scribblecrumb 
I made those changes; the idea was for each PR to have EITHER the "why it's good" section or the "linked issues" section. But I don't think it's the end of the world if the rare bugfix goes through that has neither section as long as it has the summary and proof of testing

**Comment by scribblecrumb** [comment]
I agree!  I think it's rare we have someone make a bugfix without there being an issue report about it first anyways.  Like you said, as long as they're summarizing and displaying proof of testing, it should be fine.

**Review by scribblecrumb — APPROVED** [review]

**Comment by scribblecrumb** [comment]
Oh! Sorry, just had another thought.  Do we want to add a note about proper PR titling?  (i.e. using Content, Feature, Bugfix; and giving a short, but descriptive title)

**Comment by Thlumyn** [comment]
@scribblecrumb How is that?

**Comment by scribblecrumb** [comment]
@Thlumyn Looks good!
